src/components/ButtonGroup/ButtonWrapper.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import isNil from "lodash/isNil";
+import Button from "../Button/Button";
+import Tooltip from "../Tooltip/Tooltip";
+
+export const ButtonWrapper = ({
+  tooltipContent,
+  tooltipPosition,
+  tooltipHideDelay,
+  tooltipShowDelay,
+  tooltipContainerSelector,
+  tooltipMovedBy,
+  ...otherProps
+}) => {
+  // eslint-disable-next-line react/jsx-props-no-spreading
+  let button = <Button {...otherProps} />;
+  if (!isNil(tooltipContent)) {
+    button = (
+      <Tooltip
+        moveBy={tooltipMovedBy}
+        position={tooltipPosition}
+        hideDelay={tooltipHideDelay}
+        showDelay={tooltipShowDelay}
+        content={tooltipContent}
+        showTrigger={["mouseenter"]}
```

Comment:
please extract `["mouseenter"]` and `["mouseleave"]` to constants out side the component